app/main.py

```diff
@@ -2,24 +2,62 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: type[object], name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: object, owner: type[object]) -> int:
+        value = getattr(instance, self.protected_name)
+        return value
+
+    def __set__(self, instance: object, value: int) -> ValueError | None:
+        if value < self.min_amount or value > self.max_amount:
+            raise ValueError
```

Comment:
it is better to add some message to exception it is a bad practice to use exceptions without message